notifications/service/service.go

```diff
@@ -0,0 +1,34 @@
+package service
+
+import (
+	"dont-slack-evil/apphome"
+	"log"
+	"os"
+
+	"github.com/slack-go/slack"
+)
+
+var slackBotUserOauthToken = os.Getenv("SLACK_BOT_USER_OAUTH_ACCESS_TOKEN")
```

Comment:
no you cannot use this. the token is no longer an ENV variable but must be read from the Teams database

I believe you want your function to send reports about users that have too many bad messages : you actually need to iterate on every team that's in our DB to return the teamIDs and their associated bot user oauth token, and run the logic for each of the teams